Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -118,6 +118,7 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
     renderTitleComponent: PropTypes.func,
     style: View.propTypes.style,
     statusBarHeight: PropTypes.number,
+    // $FlowFixMe(>=0.41.0)
```

Comment:
Not sure about why this error happens so $FlowFixMe seemed like an ok solution for now... Here's the full error
```
Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js:121
121:     viewProps: PropTypes.shape(View.propTypes),
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ call of method `shape`. Expected a React PropType instead of
372:     hitSlop: EdgeInsetsPropType,
                  ^^^^^^^^^^^^^^^^^^ object type. See: Libraries/Components/View/View.js:372
```